src/main/java/org/zeromq/ZContext.java

```diff
@@ -31,6 +35,11 @@
     private List<Socket> sockets;
 
     /**
+     * List of selectors managed by this ZContext
+     */
+    private List<Selector> selectors;
+
+    /**
```

Comment:
That's a good idea. ZContext actually has a Context (not a Ctx directly), but Context has a Ctx, so by extension, every ZContext has a Ctx. So, we should be able to put the list of Selectors at the "lowest level" Ctx and avoid duplicating code.

I'll take a stab at this soon when I have a minute.